README.md

```diff
@@ -5,6 +5,8 @@ This repo contains the .NET Core runtime optimized for ahead of time compilation
 
 If you would like to give CoreRT a try, we publish daily snapshots of CoreRT to a NuGet feed. Using CoreRT is as simple as adding a new package reference to your .NET Core project and publishing it. Check out one of our samples: a "[Hello World](samples/HelloWorld)" console app, a simple [ASP.NET Core](samples/WebApi/) app, a [MonoGame](samples/MonoGame/) game or a [native library](samples/NativeLibrary). The `README.md` file in each sample's directory will guide you through the process step by step.
 
+If you want to see CoreRT working with database, can see from other such as ADO.NET [ifew/corert-db](https://github.com/ifew/corert-db), EntityFrameworkCore.Sqlite [rubin55/dot-hello](https://github.com/rubin55/dot-hello)
```

Comment:
Could you please add move this to the next paragraph next to MonoGame and Avalonia links:

...
- [Avalonia](https://www.youtube.com/watch?v=iaC67CUmEXs) demo
- ADO.NET example [ifew/corert-db](https://github.com/ifew/corert-db)
- EntityFrameworkCore.Sqlite example [rubin55/dot-hello](https://github.com/rubin55/dot-hello)